changelogs/unreleased/gh-8657-move-triggers-to-trigger-registry.md

```diff
@@ -1,7 +1,7 @@
 ## feature/lua
 
-* **[Breaking change]** Triggers from `space_object` and `box.session` were
-  moved to the trigger registry (gh-8657). Now triggers set with
+* **[Breaking change]** Triggers from `space_object`, `box.session` and
+  `box.ctl` were moved to the trigger registry (gh-8657). Now triggers set with
   `space:on_replace()` or `space:before_replace()` are attached to the space id,
   not the space object. So, if you delete a space with registered triggers and
   create another space with the same id, it will use the triggers that are left
```

Comment:
```suggestion
* **[Breaking change]** Triggers from `space_object`, `box.session`, and
  `box.ctl` were moved to the trigger registry (gh-8657). Now triggers set with
  `space_object:on_replace()` or `space_object:before_replace()` are attached to the space id,
  not to `space_object`. So, if you delete a space with registered triggers and
  create another space with the same id, it will use the remaining triggers.
```